core/src/main/scala/Actionable.scala

```diff
@@ -71,15 +71,14 @@ object Actionable {
 
     def action(unit: UnitDef @@ Versioned): Kleisli[IO, (NelsonConfig,ActionConfig), Unit] =
       Kleisli { case (cfg, actionConfig) =>
-        val unitw = Manifest.Versioned.unwrap(unit)
         val ttl = cfg.cleanup.initialTTL.toSeconds
         val exp = Instant.now.plusSeconds(ttl)
         val plan = actionConfig.plan
         val dc = actionConfig.datacenter
         val ns = actionConfig.namespace
         val hash = actionConfig.hash
         val policy =
-          if (Manifest.isPeriodic(unitw, plan))
+          if (Manifest.isPeriodic(plan))
```

Comment:
Why is the unit not being passed in here?

Reply:
It just wasn't being used in the `isPeriodic` definition - all the changes in this PR besides the version bumps are I guess new scalac flags being turned on and triggering errors with unused variables/imports.